oneoffs/eval_sgf_to_cbt.py

```diff
@@ -103,7 +110,7 @@ def canonical_name(sgf_name):
 
     # Often eval is inside a folder with the run name.
     # include from folder before /eval/ if part of path.
-    with_folder = re.search(r'/([^/]*/eval/.*)', sgf_name)
+    with_folder = re.search(r'(?:^|/)([^/]*/eval/.*)', sgf_name)
```

Comment:
if you don't provide a full path this wouldn't always work.

?: is to avoid (^|/) being a capture group (and having to renumber group(1) below)